internal/e2e/testdata/test_server.txt

```diff
@@ -1,40 +1,41 @@
 queryString: test server
 query: (and substr:"test" substr:"server")
-targetRank: 5
+targetRank: 72
 
-github.com/golang/go/test/fixedbugs/issue33355.go
-38:type Server struct {
-10:package server
-20:	srv  *Server
+github.com/golang/go/src/cmd/vendor/golang.org/x/mod/sumdb/test.go
```

Comment:
I think you could argue that this is actually a good result (in the absence of other info on the query intent)? And that what we need is better file ranking to exclude vendor directories?

Reply:
good point. It is a bit dissapointing though that our target document goes to 72nd place.